plugin/trino-hive/src/main/java/io/trino/plugin/hive/gcs/GoogleGcsConfigurationInitializer.java

```diff
@@ -44,13 +45,13 @@ public void initializeConfiguration(Configuration config)
 
         if (useGcsAccessToken) {
             // use oauth token to authenticate with Google Cloud Storage
-            config.set(AUTH_SERVICE_ACCOUNT_ENABLE.getKey(), "false");
```

Comment:
nit: Can we do this method change as a prep. commit and then maybe update the library ?